content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -177,44 +177,60 @@ On the first usage of a GDrive [remote](/doc/command-reference/remote), for
-[URL](#url-format) in question. The
-[auth process](https://developers.google.com/drive/api/v2/about-auth) will ask
-you to grant DVC the necessary permissions, and produce a verification code
-needed for DVC to complete the connection. On success, the necessary credentials
-will be saved in a Git-ignored file, located in
-`.dvc/tmp/gdrive-user-credentials.json` and they will be used automatically next
-time you run DVC.
-
-⚠️ In order to prevent unauthorized access to your Google Drive, **do not share
+[URL](#url-format) in question. The [auth process] will ask you to grant DVC the
+necessary permissions, and produce a verification code needed for DVC to
+complete the connection. On success, the necessary credentials will be cached
+globally ([see `gdrive_user_credentials_file`]) and used automatically next time
+DVC needs them.
+
+<admin type="warn">
+
+In order to prevent unauthorized access to your Google Drive, **do not share
 these credentials with others**. Each team member should go through this process
 individually.
 
-If you use multiple GDrive remotes, by default they will be sharing the same
-`.dvc/tmp/gdrive-user-credentials.json` file. It can be overridden with the
-`gdrive_user_credentials_file` parameter:
+</admin>
+
+If you use multiple GDrive remotes, by default they will share the same cached
+credentials. To isolate them, you can use custom profile names for different
+remotes:
+
+```
+$ dvc remote modify --local myremote profile 'myuser'
+```
+
+You can also overwrite the cached credentials file location completely, for
+example to have it inside your project directory:
 
 ```dvc
 $ dvc remote modify myremote --local \
       gdrive_user_credentials_file .dvc/tmp/myremote-credentials.json
 ```
 
-⚠️ In order to prevent unauthorized access to your Google Drive, **never
-commit** this file with Git. Instead, add it into `.gitignore` and never share
-it with other people.
+<admin type="warn">
+
+If the file is inside a Git repo, **do not commit it** to Git in order to
+prevent unauthorized access to your Google Drive. Add it to `.gitignore` to be
+sure.
 
-If you wish to change the user you have authenticated with, or for
-troubleshooting misc. token errors, simply remove the user credentials JSON file
-and authorize again.
+</admin>
+
+To change the user you have authenticated with or for troubleshooting misc.
+token errors, you can remove the user credentials file and authorize again.
 
 Alternatively, a `GDRIVE_CREDENTIALS_DATA` can be set to pass user credentials
 in CI/CD systems, production setup, read-only file systems, etc. The content of
-this variable should be a string with JSON that has the same format as in the
-credentials files described above, and usually you get it going through the same
-authentication process. If `GDRIVE_CREDENTIALS_DATA` is set, the
-`gdrive_user_credentials_file` value (if provided) is ignored.
+this variable should be a JSON string with the same content as credentials file
```

Comment:
why touching this? is it related?

Reply:
I guess not. For a moment I thought this PR also introduced `gdrive_user_credentials_file` but no. Reverted.